CryptoPkg/Library/BaseCryptLib/Pk/CryptRsaBasic.c

```diff
@@ -112,6 +114,9 @@ RsaSetKey (
   BnDq   = NULL;
   BnQInv = NULL;
 
+  AllocatedBn[0] = NULL;

```

Comment:
Why add new param here? I think
if (BnX != NULL) {
  BN_free (BnX);
}
is good enough.

Reply:
> BnX is not always allocated since it can be retrieved from `RSA_get0_key` \ `RSA_get0_factors` \ `RSA_get0_crt_params`. Have to deallocate only if it gets initialized thru `BN_new` interface.

Agree, approved.